docker/example/supervisord/Dockerfile

```diff
@@ -10,14 +10,14 @@ LABEL version="${version}"
 ENV DEBIAN_FRONTEND=noninteractive \
     TZ=America/New_York \
     HOME=/root
-RUN (echo $TZ > /etc/timezone) \
- && dpkg-reconfigure tzdata
+RUN (echo $TZ > /etc/timezone)
```

Comment:
Also, please remove the parentheses around this command. Since you've deleted the "&& dpkg-reconfigure tzdata", the parentheses are no longer needed (if they were ever needed for anything more than clarify in the first place).

Reply:
removed the parentheses